core/frontend/src/tile/OPCFormatInterpreter.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Cartographic, EcefLocation, EcefLocationProps } from "@itwin/core-common";
+import { Range3d, Range3dProps } from "@itwin/core-geometry";
+import { ALong, CRSManager, Downloader, OnlineEngine, OPCReader, OrbitGtBounds, PageCachedFile, PointCloudReader, UrlFS } from "@itwin/core-orbitgt";
+import { FrontendLoggerCategory } from "../FrontendLoggerCategory";
+import { DownloaderNode } from "@itwin/core-orbitgt/lib/cjs/system/runtime/DownloaderNode";
+import { Logger } from "@itwin/core-bentley";
+
+const loggerCategory: string = FrontendLoggerCategory.RealityData;
+
+interface SpatialLocationAndExtents {
```

Comment:
This is a duplication of the same interface exported elsewhere.

Reply:
Remove duplicate SpatialLocationAndExtents interface definition.